extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -131,14 +132,14 @@ declare module 'dataworkspace' {
 		readonly defaultTargetPlatform?: string;
 
 		/**
-		 * Link display value for a link at the end of the project description. linkLocation also needs to be set to use this
+		 * Whether or not sdk style project is an option
 		 */
-		readonly linkDisplayValue?: string;
```

Comment:
I removed these two since they aren't used anymore. They were added for the learn more link in the SDK-style template, which has now been removed